### DISCUSSION TITLE
Error setup python-apt on Termux

### DISCUSSION THREAD

**Discussion by Sapak3** [community/community#44795] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Report is here:

https://github.com/Sapak3/ubiquitous-sapeque/files/10458508/python-apt-error.txt

Don't know who is concerned, sorry.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Sapak3 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
